freesound/settings.py

```diff
@@ -207,7 +224,7 @@
 # -------------------------------------------------------------------------------
 # Freesound miscelaneous settings
 
-SUPPORT = ()
+SUPPORT = (('Support', 'admin@example.org'), )
```

Comment:
I don't like using example.org here, can we change it? Is it needed for the tests? If so, better that we put it in test settings

Reply:
actually this is not for tests (in tests `SUPPORT` is already defined). this is for running locally, because otherwise sending to support fails (which only happens in the contact form). I'll add it to `local_settings.example.py` instead